lib/src/components/NxToggle/NxToggle.tsx

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { forwardRef } from 'react';
+import classnames from 'classnames';
+
+import './NxToggle.scss';
+import { Props, propTypes } from './types';
+export { Props } from './types';
+
+/**
+ * A stateless component for rendering a toggle control with a label
+ * @param props.toggleId An id to identify the toggle
+ * @param props.onChange A callback for when the toggle control is toggled
+ * @param props.isChecked Whether the toggle should be rendered as on/checked or off/unchecked
+ * @param props.disabled Whether the toggle should be rendered as disabled or not. When disabled, the onChange
+ * callback will not fire
+ * @param props.children VDOM rendered as label. Should be
+ * [phrasing content](https://www.w3.org/TR/2011/WD-html-markup-20110525/terminology.html#phrasing-content).
+ */
```

Comment:
We've stopped doing these comments; the gallery docs are the sole source of documentation.  These just tend to get out of date.

Reply:
> We've stopped doing these comments; the gallery docs are the sole source of documentation. These just tend to get out of date.

Removed here: https://github.com/sonatype/sonatype-react-shared-components/pull/250/commits/c5aaa128a11565cfd50fb9fcb16777f3d0868fde